tests/e2e/cluster_downgrade_test.go

```diff
@@ -37,23 +37,47 @@ import (
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
+type CancellationState int
+
+const (
+	NoCancellation CancellationState = iota
+	CancelRightBeforeEnable
+	CancelRightAfterEnable
```

Comment:
There is no need to expose them if they are just used locally.
```suggestion
	noCancellation CancellationState = iota
	cancelRightBeforeEnable
	cancelRightAfterEnable
```

Reply:
Thanks @ahrtr
Compilation errors fixed